package.json

```diff
@@ -38,6 +38,8 @@
     "@babel/preset-react": "^7.10.4",
     "babel-jest": "^26.3.0",
     "babel-loader": "^8.1.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.3",
```

Comment:
There's a table at [this link](https://github.com/enzymejs/enzyme) that suggests the enzyme-adapter-react-16 versions compatible with specific react versions. You may want to check that (I haven't spent much time on it).

Reply:
Hey, thanks Elvin. Yeah, I saw that too, somewhere in trying to make enzyme work. I'm using React 16.13, I think. So I'm good here. Thanks for the heads up!